website/docs/components/button-set/partials/code/how-to-use.md

```diff
@@ -1,8 +1,42 @@
 ## How to use this component
 
+The basic invocation requires two or more buttons to be provided as children:
+
 ```handlebars
 <Hds::ButtonSet>
   <Hds::Button @text="Submit" type="submit" />
   <Hds::Button @text="Cancel" @color="secondary" @href="https://hashicorp.com" />
 </Hds::ButtonSet>
 ```
+
+### Equal width buttons
+
+If you want to have buttons with equal width, apply a width (via inline style or via CSS class) to the `ButtonSet` container, and the argument `@isFullWidth={{true}}` to the `Button` elements:
+
+```handlebars
+<Hds::ButtonSet {{style width="15rem"}}>
+  <Hds::Button @text="Save" @isFullWidth={{true}} />
+  <Hds::Button @text="Cancel" @color="secondary" @href="https://hashicorp.com" @isFullWidth={{true}} />
+</Hds::ButtonSet>
+```
+
+#### With loading state
+
+This technique is useful if you need to show a loading state, to avoid the resizing and shifting of the buttons:
```

Comment:
```suggestion
If you need to show a loading state, this technique is useful to avoid the resizing and shifting of the buttons:
```

Reply:
With "this technique" I was referring to the technique (maybe I should use the word "approach") described in the previous example. In that case does your suggestion still apply?